draft-ietf-rats-corim.md

```diff
@@ -1429,9 +1429,155 @@ is extended using Endorsements etc. from the accepted tags.
 
 ### Matching Evidence against Reference Values
 
+This section describes the process performed by the Verifier to determine
+which Endoresments in the candidate CoMIDs are applicable to the Attester being
+verified. All applicable Endorsements are added to the Accepted Claims Set.
+
+The verifier checks whether Endorsements are applicable by comparing Evidence
+in the Accepted Claims Set against Reference Values from the CoMID. These
+Reference Values may be provided as Reference Value Triples or may be combined
+with the Endorsements, for example as the Conditional Endorsement Series Triple.
+
+The following subsections describe how the CoRIM tells the verifier which
+Reference Values and Endorsements are grouped together ({{sec-grouping-ref-vals}})
+and how the verifier matches a Reference Value against the Accepted Claims Set
+({{sec-match-one-ref-val}}).
+
+If all Reference Values in a group match the Accepted Claims Set then all
+Endorsements in the group are added to the Accepted Claims Set
+(see {{sec-add-to-acs}}).
+
+If any Reference Value in a group does not match then this does not affect
+the processing of other groups.
+
+#### Grouping Reference Values and Endorsements {#sec-grouping-ref-vals}
+
+> This paragraph will be replaced by a description of how the CoRIM tells the
+verifier which Reference Values and Endorsements are grouped together.
+
+[^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/TBD
+
+[^issue]: Need to describe how to match conditional endorsements. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/80
+
+#### Matching a Reference Value against the Accepted Claims Set {#sec-match-one-ref-val}
+
+This section describes how a Reference Value is matched against the Accepted Claims
+Set. If any part of the processing indicates that the Reference Value does not match
+then the remaining steps in this section are not required.
+
+A Reference Value consists of an `environment-map` plus a `measurement-map`. In the
+Reference Value Triple these are packaged together. In other triples multiple
+Referenve Values are stored more compactly by letting one `environment-map`
+apply to multiple `measurement-map`s.
+
+The Verifier first looks for entries in the Accepted Claims Set with the same
+`environment-map` as the reference value. If there are no such entries then
+the Reference Value does not match.
+
+A Verifier SHALL compare two `environment-map`s using a binary comparison of the CBOR
```

Comment:
Are we sure how this comparison is mentioned here ? Are we saying we compare the CBOR Encoded objects.

My understanding was that based on a specific `attestation scheme` the Environment Structure would be populated and one would compare it matching identical elements in the reference-value-triple-record.